### PR TITLE
Add missing object to api conversions

### PR DESCRIPTION
**Pull request by gclx**

### CODE REVIEW

File: Crm/Company.php [hunk 1]
```diff
@@ -719,6 +740,21 @@ public function toArrayForApi()
         if ($this->getWebsite()) {
             $return['website'] = $this->getWebsite();
         }
+        if ($this->getBic()) {
+            $return['bic'] = $this->getBic();
```
Comment (carakas) [inline Crm/Company.php]: This is not a valid option according to the api docs are you sure this works?
http://apidocs.teamleader.eu/crm.php

File: Crm/Company.php [hunk 2]
```diff
@@ -719,6 +740,21 @@ public function toArrayForApi()
...
+        }
+        if ($this->getFax()) {
+            $return['fax'] = $this->getFax();
+        }
+        if ($this->getIban()) {
+            $return['iban'] = $this->getIban();
```
Comment (carakas) [inline Crm/Company.php]: This is not a valid option according to the api docs are you sure this works?
http://apidocs.teamleader.eu/crm.php

File: Crm/Contact.php [hunk 3]
```diff
@@ -701,6 +701,21 @@ public function toArrayForApi()
                 $return['custom_field_' . $fieldID] = $fieldValue;
             }
         }
+        if ($this->getBic()) {
+            $return['bic'] = $this->getBic();
```
Comment (carakas) [inline Crm/Contact.php]: This is not a valid option according to the api docs are you sure this works?
http://apidocs.teamleader.eu/crm.php

File: Crm/Contact.php [hunk 4]
```diff
@@ -701,6 +701,21 @@ public function toArrayForApi()
...
+        }
+        if ($this->getFax()) {
+            $return['fax'] = $this->getFax();
+        }
+        if ($this->getGsm()) {
+            $return['gsm'] = $this->getGsm();
+        }
+        if ($this->getIban()) {
+            $return['iban'] = $this->getIban();
```
Comment (carakas) [inline Crm/Contact.php]: This is not a valid option according to the api docs are you sure this works?
http://apidocs.teamleader.eu/crm.php

**Comment by gclx** [comment]
@carakas Fixed

**Comment by carakas** [comment]
@gclx released in 1.10.2

**Pull request merged**